lib/typeahead_search.rb

```diff
@@ -20,13 +20,11 @@ def xapian_search
     ActsAsXapian.query_parser.default_op = Xapian::Query::OP_OR
     begin
       xapian_search = run_query
-    rescue RuntimeError => e
-      if e.message =~ /^QueryParserError: Wildcard/
-        # Wildcard expands to too many terms
-        Rails.logger.info "Wildcard query '#{query}' caused: #{e.message.force_encoding('UTF-8')}"
-        @wildcard = false
-        xapian_search = run_query
-      end
+    rescue ActsAsXapian::UnhandleRuntimeError => e
+      # Wildcard expands to too many terms
+      Rails.logger.info "Wildcard query '#{query}' caused: #{e.message.force_encoding('UTF-8')}"
```

Comment:
Wondering whether this should be `warn` rather than `info`? Not that it would make much difference to our visibility at the moment, but maybe it might be a way to keep more of an eye on this type of problem that only shows itself over time.